src/interface_adapter/search/SearchController.java

```diff
@@ -0,0 +1,24 @@
+package interface_adapter.search;
+
+import java.time.Instant;
+import use_case.search.SearchInputBoundary;
+import use_case.search.SearchInputData;
+
+public class SearchController {
+
+  private final SearchInputBoundary searchInteractor;
+
+  public SearchController(SearchInputBoundary searchInteractor) {
+    this.searchInteractor = searchInteractor;
+  }
+
+  public void executeSearchRequest(String roomID, String message, String authID) {
```

Comment:
You can get rid of writing execute here if you want and just write `searchRequest` instead